portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +36,17 @@ function addRandomIntro() {
   // Add it to the page.
   const introContainer = document.getElementById('intro-container');
   introContainer.innerText = intro;
+}
+
+async function getRandomQuoteUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('quote-container').innerText = quote;
+}
+
+function loadTasks() {
+    let list = fetch('/list-tasks').then(response => response.json());
+    for(c in list){ 
+        document.getElementById("reviews").appendChild(document.createElement("DIV").appendChild(document.createTextNode(c.type.innerHTML+=c.description.outerHTML)));
+    }
```

Comment:
Divide this out into variables to make it easier to understand.

let reviewsElem = document.getElementById("reviews");
let commentElem = document.createElement("DIV");
commentElem.appendChild(document.createTextNode(c.type.innerHTML+c.description.outerHTML));

I think the issue here is that you did an assignment instead of passing in text to document.createTextNode(...);
c.type.innerHTML+=c.description.outerHTML (isn't text; since you're just re-assigning c.type.innerHTML)
c.type.innerHTML+c.description.outerHTML (is text; i think this is what you're looking for)

Reply:
And then don't forget to appendChild to the reviewsElem at the end.

reviewsElem.appendChild(commentElem)